zipline/data/history_loader.py

```diff
@@ -343,6 +358,20 @@ def _calendar(self):
     def _array(self, start, end, assets, field):
         pass
 
+    def _decimal_places_for_asset(self, asset, reference_date):
+        if isinstance(asset, Future) and asset.tick_size:
+            return number_of_decimal_places(asset.tick_size)
+        elif isinstance(asset, ContinuousFuture):
+            rf = self._roll_finders[asset.roll_style]
+            contract_sid = rf.get_contract_center(
+                asset.root_symbol, reference_date, asset.offset,
+            )
```

Comment:
Yeah the offset doesn't matter. Then this way I don't need to set a `_roll_finders` attribute either.